packages/vue-renderer/src/renderers/spa.js

```diff
@@ -13,27 +12,21 @@ export default class SPARenderer extends BaseRenderer {
 
     this.cache = new LRU()
 
-    // Add VueMeta to Vue (this is only for SPA mode)
-    // See app/index.js
-    Vue.use(VueMeta, {
+    this.vueMetaConfig = {
       keyName: 'head',
       attribute: 'data-n-head',
       ssrAttribute: 'data-n-head-ssr',
+      ssrAppId: '1',
```

Comment:
Number (`1`) implies multiple SSR apps are possible. Is that a case?

Reply:
Actually the `'1'` implies that we expect that the Nuxt SPA app is always the first APP on the page (I made it a string on purpose as a small hint).

The term `ssrAppId` might be a bit confusing here, because in SPA mode there is no ssr app but we still need to make sure the head config from nuxt.config is recognised once the page is loaded on the client.

Reply:
Hmm why not simply naming it `nuxt` or `main` (like `__nuxt`)

Reply:
That would be fine if I would know there would only ever be one app using VueMeta on the page, but unfortunately I dont know that.

As mentioned in my edit, this boils down to a hydration issue as I cant reliably pass an appId from ssr to the client. We could maybe support it for Nuxt if we just add an additional attribute in the app template.

Fyi, [this](https://github.com/vuejs/vue/blob/e90cc60c4718a69e2c919275a999b7370141f3bf/packages/vue-server-renderer/basic.js#L8757) is were Vue sets the data-server-rendered attribute and afaik I cant do something similar (through Vue)

Reply:
Let me know if you want me to take the route of adding a vue-meta specific attribute on the APP div. Thinking about it, it should be possible to just remove that ssrAttribute.